src/app/code/community/Shopgate/Cloudapi/Model/Frontend/Observer/Layout.php

```diff
@@ -21,6 +21,16 @@
  */
 class Shopgate_Cloudapi_Model_Frontend_Observer_Layout
 {
+    /**
+     * Define custom handles
+     */
+    const CUSTOM_HANDLES = array(
```

Comment:
I think php5.3/5.4 does not allow this kind of stuff, use property instead

Reply:
fixed